app/src/main/java/com/sdp/swiftwallet/presentation/main/SettingsActivity.java

```diff
@@ -0,0 +1,27 @@
+package com.sdp.swiftwallet.presentation.main;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+import com.sdp.cryptowalletapp.R;
+
+/**
+ * Settings Activity
+ */
+public class SettingsActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_settings);
+
+        if (findViewById(R.id.settings_framelayout) != null) {
+            if (savedInstanceState != null) {
+                return;
+            }
+            // below line is to inflate our fragment.
+            getFragmentManager().beginTransaction().add(R.id.settings_framelayout, new SettingsFragment()).commit();
```

Comment:
I have no idea if you plan to have more fragment than this to your manager but if it is the case you might consider doing a navigation graph, it automates a lot of stuff. Just in case

Reply:
We shouldn't need any more fragments for the settings. For some reason the way of making settings screens is super weird in Android but you basically need to have an empty Activity with inside an Empty Fragment and then you can use this other xml definition for a list of settings options.